tests/integration/terraform/test_terraform.tf/.terraform-version

```diff
@@ -0,0 +1 @@
+0.11.13
```

Comment:
Can we extend this to test multiple Terraform versions (e.g. so we can test both 0.11.13 and 0.12.6)?

Reply:
added test for terraform version